crates/oxc_linter/src/rules/typescript/no_explicit_any.rs

```diff
@@ -0,0 +1,643 @@
+use crate::Fix;
+use oxc_ast::AstKind;
+use oxc_diagnostics::{
+    miette::{self, Diagnostic},
+    thiserror::Error,
+};
+use oxc_macros::declare_oxc_lint;
+use oxc_span::Span;
+use serde_json::Value;
+
+use crate::{context::LintContext, rule::Rule, AstNode};
+
+#[derive(Debug, Error, Diagnostic)]
+#[error("Unexpected any. Specify a different type.")]
```

Comment:
Need to change this to `typescript-eslint(no-explict-any):` and update the snapshot.

Reply:
fixed